README.md

```diff
@@ -146,6 +146,12 @@ Specify a config object to use instead of what exists locally
 ember-template-lint --config '{ "rules": { "no-implicit-this": { "severity": 2, "config": true } } }' test/fixtures/no-implicit-this-allow-with-regexp/app/templates
 ```
 
+Specify a config file to run a certain set of rules
```

Comment:
I think we need to tweak this a tad, this isn't specifying a custom config file (which is what the prose says). You'd do that by `--config-path` option. 

I think you mean:

```suggestion
Specify a configuration to override the normally loaded config file:
```